depfinder/pkg_data/pkg_data.yml

```diff
@@ -0,0 +1,19 @@
+_PACKAGE_MAPPING:
+  av: pyav
+  bs4: beautifulsoup4
+  cv2: opencv
+  IPython: ipython
+  netCDF4: netcdf4
+  PIL: pillow
+  skimage: scikit-image
+  sklearn: scikit-learn
+  stdlib_list: stdlib-list
+  yaml: pyyaml
+
+_NAMEDSPACE_MAPPING:
+  ruamel.yaml: ruamel.yaml
+
+
+_FAKE_PACKAGES:
+  matplotlib: !!set {mpl_toolkits: null}
```

Comment:
I don't really like the `!!set {value: null}` syntax but I'm not a `yaml` expert to figure out a better way. Maybe we can change the logic in `depfinder` to take lists there if that makes sense. (But I would leave that for another PR :smile:)

Reply:
I agree that this is kind of gross syntax. I also agree that changing this to lists can wait for another PR.